app/storage/chroma.py

```
@@ -0,0 +1,197 @@
+from app.utils.logging_config import get_logger
+
+logger = get_logger(__name__)
+
+def init_chroma():
+    """
+    Initialize ChromaDB client and collection.
+    
+    Returns:
+        ChromaDB client with initialized collection
+    """
+    host = os.getenv("CHROMA_HOST", "localhost")
```

Ruff (0.8.2)

Local variable `host` is assigned to but never used

Remove assignment to unused variable `host`

(F841)